dispatch/static/manager/src/js/pages/LoginPage.js

```diff
@@ -32,19 +34,35 @@ class LoginPageComponent extends React.Component {
 
   render() {
     return (
-      <div>
-        <form onSubmit={this.onSubmit}>
+      <div className='c-login-page'>
+        <form onSubmit={this.onSubmit} className='pt-card pt-elevation-2'>
+          <div className='c-login-page__heading'>
+            Dispatch Login
+          </div>
           <input
+            className='pt-input pt-round'
             name="email"
             type="email"
+            placeholder='Email'
             value={this.state.email}
             onChange={this.onChangeEmail} /><br/>
           <input
+            className='pt-input pt-round'
             name="password"
             type="password"
+            placeholder='Password'
             value={this.state.password}
             onChange={this.onChangePassword} /><br/>
-          <input type="submit" />
+          <button
+            className='pt-button pt-large'
```

Comment:
The button seems a bit big compared to the text inputs -- maybe just use the regular size?